package.json

```diff
@@ -64,37 +65,38 @@
   "devDependencies": {
     "@commitlint/cli": "17.2.0",
     "@commitlint/config-conventional": "17.2.0",
+    "@cypress/grep": "3.1.4",
```

Comment:
this shouldn't have come back I don't believe @radoslaw-szpiech to confirm

Reply:
Agree, we don't use that in the project at the moment.